src/_listDefinitions.scss

```diff
@@ -40,6 +40,27 @@ $fd-list-navigation-navigator-bar-right: inset -0.1875rem 0 0 0 var(--sapSelecte
 $fd-list-navigation-navigator-bar-left: inset 0.1875rem 0 0 0 var(--sapSelectedColor);
 $fd-list-navigation-indicator-width: 2.75rem !default;
 
+$fd-list-optional-text-font-family: var(--sapUiFontFamily);
+$fd-list-optional-text-font-size: var(--sapMFontMediumSize);
+$fd-list-optional-text-font-weight: normal;
+$fd-list-optional-text-font-color: var(--sapUiContentLabelColor);
+$fd-list-object-border-bottom: 1px solid var(--sapList_BorderColor);
+$fd-list-object-with-value-zero: 0;
+$fd-list-object-with-value-line-height: 1.4;
+$fd-list-object-with-padding-value-one: 1rem;
+$fd-list-object-with-padding-value-three: 3rem;
+$fd-list-object-with-vertical-align-top: top;
+$fd-list-object-with-vertical-align-bottom: bottom;
+$fd-list-object-attribute-font-family: var(--sapFontFamily);
+$fd-list-object-font-size: var(--sapFontSize);
+$fd-list-object-font-weight: normal;
+$fd-list-object-attribute-font-color: var(--sapContent_LabelColor);
+$fd-list-object-number-font-family: var(--sapFontLightFamily);
+$fd-object-identifier-title-font-family: var(--sapFontHeaderFamily);
+$fd-object-identifier-title-font-size: var(--sapFontHeader4Size);
+$fd-object-identifier-title-font-weight: var(--sapFontHeaderWeight);
+$fd-object-identifier-title-color: var(--sapGroup_TitleTextColor);
```

Comment:
A lot of these variables seem unnecessary - can you do an audit that they are 1) the correct variables and 2) that they are actually needed and not already set by `fd-reset()`

Reply:
where ever usage of fd-reset and fd-typography is not causing issues updated it as well and removed un required variables.